clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -203,7 +206,7 @@ public void testInvalidSocketReceiveBufferSize() {
 
     @Test
     public void testSubscription() {
-        KafkaConsumer<byte[], byte[]> consumer = newConsumer();
+        KafkaConsumer<byte[], byte[]> consumer = newConsumer("");
```

Comment:
nit: not sure if there's a strong reason for any of these to use the empty group id. Could we use `groupId` instead?